src/Microsoft.OpenApi.Readers/V3/OpenApiV3VersionService.cs

```diff
@@ -129,6 +129,20 @@ public OpenApiV3VersionService(OpenApiDiagnostic diagnostic)
                         }
                         id = localSegments[3];
                     }
+                    else if (id.StartsWith("/paths/"))
+                    {
+                        var localSegments = segments[1].Split('/');
+                        if (localSegments.Length == 3)
+                        {
+                            // The reference of a path may contain JSON escape character ~1 for the forward-slash character, replace this otherwise
+                            // the reference cannot be resolved.
+                            id = localSegments[2].Replace("~1", "/");
```

Comment:
I get error message `No overload for method 'Replace' takes 3 arguments`, it seems this library's target framework is netstandard2.0, and for that is only defined:
```
System.String.Replace(String, String);
System.String.Replace(char, char);
```
What am I missing?

Reply:
Ah right, it might only be available with the newer versions. Can you switch the parameter to an array please? Make it a constant so we don't alloc the array on each call. Thanks!

Reply:
` switch the parameter to an array` are you referring to the previous comment for split? There is no option for the `replace` to take a parameter as array, the only definitions for replace I could find were the 2 I mention above.

Reply:
Sorry, I mixed things up. Can you switch to chars please? Forget the comparison. Thanks

Reply:
The string "~1" cannot be represented as a character, unfortunately I get the compile error "Too many characters in character  literal".

Reply:
thank you for your patience with me. I was reviewing this on a phone. Resolving.